com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -525,6 +525,13 @@ private void SpawnNetworkObjectLocallyCommon(NetworkObject networkObject, ulong
                 triggerInfo.TriggerData.Dispose();
                 m_Triggers.Remove(networkId);
             }
+
+            // propagate the IsSceneObject setting to child NetworkObjects
```

Comment:
is this a fix for scene objects only or does it also fix NetworkPrefabs spawned at runtime too?